### PR TITLE
As18 upgrade

### PR DESCRIPTION
**Pull request by jtenner**

### CODE REVIEW

File: packages/cli/src/run.ts [hunk 1]
```diff
@@ -611,7 +595,10 @@ export function run(cliOptions: Options, compilerArgs: string[]): void {
       }
 
       Promise.all(filePromises).then(() => {
-        if (failed) process.exit(1);
+        if (failed) {
+          console.log(errors);
```
Comment (willemneal) [inline packages/cli/src/run.ts:599]:
```suggestion
          console.error(errors);
```

**Review by willemneal — APPROVED** [review]
LGTM just one nit!

File: packages/cli/src/run.ts (hunk 1 above)
Comment (jtenner) [reply packages/cli/src/run.ts:599]: Shouldn't we remove this altogether? Let's fix this right now. 

What's the best way to report these errors to stderr? Is that enough?

Comment (willemneal) [reply packages/cli/src/run.ts:599]: I think it's enough.  Though perhaps each error should be sent to console.error.  So `errors.map(console.error)`? Perhaps that prints better?

**Pull request merged**